js/repl/loadBundle.js

```diff
@@ -70,23 +79,26 @@ export default async function loadBabel(
   // See if a released version of Babel was passed
   // Prod (with URL rewriting): /repl/version/1.2.3/
   // Dev: /repl/#?version=1.2.3
-  let version = config.version;
-
-  const versionFromPath = window.location.pathname.match(/\/version\/(.+)\/?$/);
+  let version = state.version;
+  const versionKey = config.versionKey || "version";
+  const versionRegExp = new RegExp(`/${versionKey}\/(.+)\/?$`);
+  const versionFromPath = window.location.pathname.match(versionRegExp);
   if (versionFromPath) {
     version = versionFromPath[1];
   }
 
   // No specific version passed, so just download the latest release.
   if (!version) {
-    version = DEFAULT_BABEL_VERSION;
+    version = config.version;
   }
 
-  const babelStandalone =
+  const base = config.baseUrl;
+
+  const packageName =
     (semver.valid(version) && semver.gte(version, "7.0.0-beta.4")) ||
     version >= 7
-      ? "@babel/standalone"
```

Comment:
my thoughts: I kinda feel like it's easier to read/maintain with hardcoding this stuff? If we rename or anything like that I'm not sure I'd know where to look in this codebase with so much indirection? And in this case we can't substitute babel-standalone env in the same way because it won't be published until beta.33 not beta.4